docs/api/users-get-user-by-email.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+---
+
+# Get user by Email
+
+Returns an array of  [`user`](user) objects that contains only the user specified by the `email` parameter, if it exists.
+
+## URL
+
+```shell
+
+{server_url}/users/{email}
```

Comment:
```suggestion
{server_url}/users?email=<email address to find>
```
Check to see if this request requires a query parameter instead of a URL parameter.